src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/References/UnSetAttributeCommand.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.References
+{
+    internal class UnSetAttributeCommand : IProjectSystemUpdateReferenceOperation
+    {
+        private readonly ConfiguredProject _selectedConfiguredProject;
+        private readonly string _itemSpecification;
+        private readonly AbstractReferenceHandler _referenceHandler;
+
+        public UnSetAttributeCommand(AbstractReferenceHandler abstractReferenceHandler, ConfiguredProject selectedConfiguredProject, string itemSpecification)
+        {
+            _referenceHandler = abstractReferenceHandler;
+            _selectedConfiguredProject = selectedConfiguredProject;
+            _itemSpecification = itemSpecification;
+        }
+
+        public async Task<bool> ApplyAsync(CancellationToken cancellationToken)
+        {
+            IProjectItem item = await GetProjectItem();
+
+            if (item != null)
+            {
+                await item.Metadata.SetPropertyValueAsync(ProjectReference.TreatAsUsedProperty, PropertySerializer.SimpleTypes.ToString(false), null);
+            }
```

Comment:
If `item` is null, should this return `false` or flag an error somewhere? Ditto below in `RevertAsync`, and twice again in `SetAttributeCommand`. The only false return I see is in the null command.

Reply:
Done